dts/nxp/kinetis/MKE15Z256VLH7-pinctrl.h

```diff
@@ -0,0 +1,304 @@
+/*
+ * NOTE: Autogenerated file by gen_soc_headers.py
+ * for MKE15Z256VLH7/signal_configuration.xml
+ *
```

Comment:
These headers need a copyright, there should be a `--copyright` commandline switch you can add when generating them